jumpscale/packages/admin/frontend/components/solutions/Info.vue

```diff
@@ -36,7 +36,7 @@ module.exports = {
         cancelSolution: false,
       },
       KeysWithTypeList: ["Node ids", "wids", "Active workload ids"],
-      KeysWithTypeDict: ["nodes"]
+      KeysWithTypeDict: ["nodes", "Volumes"]
```

Comment:
Should it be Capitalized or lowercase like nods

Reply:
the key returned as it is from the backend, most the keys are capitalized except nodes and we are not sure where it is used.